src/blocks/separator/Separator.types.ts

```diff
@@ -1,61 +1,34 @@
-import { CSSProperties } from 'react';
-
-import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
 import { FlattenSimpleInterpolation } from 'styled-components';
 
 export type SeparatorResponsiveProps = {
+  /* Sets height css property */
   height?: ResponsiveProp<string>;
   /* Sets margin css property */
   margin?: ResponsiveProp<SpaceType>;
-  /* Sets max-height css property */
-  maxHeight?: ResponsiveProp<string>;
-  /* Sets min-height css property */
-  minHeight?: ResponsiveProp<string>;
-  /* Sets max-width css property */
-  maxWidth?: ResponsiveProp<string>;
-  /* Sets min-width css property */
-  minWidth?: ResponsiveProp<string>;
-  /* Sets padding css property */
-  padding?: ResponsiveProp<SpaceType>;
   /* Sets width css property */
   width?: ResponsiveProp<string>;
 };
 
 export type SeparatorNonResponsiveProps = {
```

Comment:
Let's remove SeparatorNonResponsiveProps as it's not required and combine it's props into SeparatorComponentProps